src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -66,13 +68,18 @@ const VoteReferendum = ({ className, referendumId, onAccountChange, lastVote, se
 	const [loginWallet, setLoginWallet] = useState<Wallet>();
 	const [availableBalance, setAvailableBalance] = useState<BN>(ZERO_BN);
 	const [balanceErr, setBalanceErr] = useState('');
+	const [successModal,setSuccessModal] = useState(false);
 	const [splitForm] = Form.useForm();
 	const [abstainFrom] = Form.useForm();
 	const[ayeNayForm] = Form.useForm();
 	const [abstainVoteValue, setAbstainVoteValue] = useState<BN>(ZERO_BN);
 	const [ayeVoteValue, setAyeVoteValue] = useState<BN>(ZERO_BN);
 	const [nayVoteValue, setNayVoteValue] = useState<BN>(ZERO_BN);
 	const [walletErr, setWalletErr] = useState<INetworkWalletErr>({ description: '', error: 0, message: '' });
+	const[ayeVoteVal,setAyeVoteVal] = useState<BN>(ZERO_BN);
+	const[nayVoteVal,setNayVoteVal] = useState<BN>(ZERO_BN);
```

Comment:
use pervious ayeVoteValue ,nay......... values, instead of creating new

Reply:
resolved